manual/docker/Developer-Edition/docker-compose.yml

```diff
@@ -37,7 +37,7 @@ services:
       - DB_ROOT_PASSWD=PASSWORD  # Requested, the value should be root's password of MySQL service.
       - SEATABLE_SERVER_LETSENCRYPT=False # Default is False. Whether to use let's encrypt certificate.
       - SEATABLE_SERVER_HOSTNAME=example.seatable.com # Specifies your host name.
-      - TIME_ZONE=Etc/UTC # Optional, default is UTC. Should be uncomment and set to your local time zone.
+      - TIME_ZONE=Etc/UTC # Optional, default is UTC. Example: Europe/Berlin. Choices can be found here: http://en.wikipedia.org/wiki/List_of_tz_zones_by_name  
```

Comment:
Wikipedia can only be used for informational purposes, it can't be the reference for this setting. Please clarify the timezone database. And the link should be to the reference documentation of the library / implementation with which this value is used as configuration input.

Reply:
Postponed.

Reply:
The wikipedia link is copied from the dtable_web_settings.py optional settings: https://manual.seatable.io/config/dtable_web_settings/